projects/wave-core-new/src/lib/operators/dialogs/expression-operator/expression-operator.component.html

```diff
@@ -41,7 +41,7 @@
 
         <mat-form-field>
             <mat-label>No-Data Value</mat-label>
-            <input matInput type="number" formControlName="noDataValue" />
+            <input matInput type="text" formControlName="noDataValue" />
```

Comment:
Why text?

Reply:
s.t. users can set NAN as the no-data value. That was the whole point of the task... Now they can either write a number or "nan" in order to create the layer

Reply:
Hm, I see. We could allow NAN only for floats and we should then present a checkbox for NAN.
This box could disable the number field then.
What do you think?

Reply:
that would also be possible. however the way I did it is also fine imho.